frontend/src/util/LiveMessageManager.js

```diff
@@ -61,6 +62,13 @@ export class LiveMessageManager {
       this.broadcastChannel.postMessage({ status: this.status })
     }
 
+    if (this.eventSource && (message.status || message.tlm)) {
+      // There are two leaders.
+      if (message.fromTabId > this.tabId) {
```

Comment:
Is this related to the master election? Is the strategy described somewhere (centralised) in the code? If not it would be nice to describe the "algorithm".

For example, here: you have two leaders. Ok. There is a message coming from a tabId "greater" than the current tabId. Why do this lead to a closeConnection()?